tests/parsed_finite_element_04.cc

```diff
@@ -19,9 +19,9 @@
 int main ()
 {
   initlog();
-  ParsedFiniteElement<2> fe_builder("FE", "FESystem[FE_Q(2)^d-FE_DGP(1)]", "u", 1);
+  ParsedFiniteElement<2> fe_builder("FE", "FESystem[FE_Q(2)^d-FE_DGP(1)]", "u,u,p", 1);
```

Comment:
Why did you correct this? This test is checking that it actually fails when it is supposed to... (`stage=run`)...